cmd/simulator/README.md

```diff
@@ -0,0 +1,87 @@
+# Load Simulator
+
+When building developing your own blockchain using `subnet-evm`, you may want to analyze how your fee parameterization behaves and/or how many resources your VM uses under different load patterns. For this reason, we developed `cmd/simulator`. `cmd/simulator` lets your drive arbitrary load across any number of [endpoints] with a user-specified `keys` directory (insecure) `timeout`, `concurrency`, `base-fee`, and `priority-fee`.
+
+## Building the Load Simulator
+
+To build the load simulator, navigate to the base of the simulator directory:
+
+```bash
+cd $GOPATH/src/github.com/ava-labs/subnet-evm/cmd/simulator
+```
+
+Build the simulator:
+
+```bash
+go build -o ./simulator *.go
+```
+
+To confirm that you built successfully, run the simulator and print the version:
+
+```bash
+./simulator -v
+```
+
+This should give the following output:
+
+```
+v0.0.1
+```
+
+To run the load simulator, you must first start an EVM based network. The load simulator works on both the C-Chain and Subnet-EVM, so we will start a single node network and run the load simulator on the C-Chain.
+
+To start a single node network, follow the instructions from the AvalancheGo [README](https://github.com/ava-labs/avalanchego#building-avalanchego) to build from source.
+
+Once you've built AvalancheGo, open the AvalancheGo directory in a separate terminal window and run a single node non-staking network with the following command:
+
+```bash
+./build/avalanchego --staking-enabled=false --network-id=local
+```
+
+:::warning
+The staking-enabled flag is only for local testing. Disabling staking serves two functions explicitly for testing purposes:
+
+1. Ignore stake weight on the P-Chain and count each connected peer as having a stake weight of 1
+2. Automagically opts in to validate every Subnet
```

Comment:
```suggestion
2. Automatically opts in to validate every Subnet
```

Reply:
I did this intentionally, but will update it